modules/users/client/components/ProfileViewBasics.js

```diff
@@ -133,6 +133,20 @@ export default function ProfileViewBasics({ profile }) {
               )
             );
           })}
+        {/* Couchers.org */}
+        {profile.extSitesCouchers && (
+          <li className="social-profile">
+            <i className="social-profile-icon icon-fw icon-lg icon-bw"></i>
```

Comment:
This icon will need updating:

<img width="339" alt="Screenshot 2022-01-05 at 20 31 53" src="https://user-images.githubusercontent.com/87168/148271244-fd621aa3-9624-4c19-ba94-57853b5c41c7.png">

Reply:
Ah good point, thanks

Reply:
Fixed